src/App.module.scss

```diff
@@ -1,38 +1,41 @@
-.App {
-  text-align: center;
-}
+@import url('https://fonts.googleapis.com/css2?family=Orbitron&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Orbitron&family=Rajdhani&display=swap');
 
-.appLogo {
-  height: 40vmin;
-  pointer-events: none;
+@mixin title($textcolor: white) {
+  font-family: 'Orbitron', sans-serif;
+  font-size: 26px;
+  color: $textcolor;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .appLogo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+@mixin blackTitle($textcolor) {
+  @include title(black);
 }
 
-.appHeader {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+.subtitles {
```

Comment:
I think you'll want this to be a mixin too - otherwise it will be really tricky to use it!

(same for the other classes here)